src/deploy/functions/params.ts

```diff
@@ -636,9 +744,36 @@ async function promptSelect<T extends RawParamValue>(
     }),
   });
   const converted = converter(response);
-  if (typeof converted === "object") {
+  if (shouldRetry(converted)) {
     logger.error(converted.message);
     return promptSelect<T>(prompt, input, resolvedDefault, converter);
   }
   return converted;
 }
+
+async function promptSelectMultiple<T extends string>(
+  prompt: string,
+  input: ListSelectInput,
+  resolvedDefault: T[] | undefined,
+  converter: (res: string[]) => T[] | retryInput
+): Promise<T[]> {
+  const response = await promptOnce({
```

Comment:
In firebase init we explain to people that they should use space to select a list item and return to choose the selection. We should probably add that to the prompt here. Similarly, when using a text input for list items, we should explain the delimiter usage.

Reply:
+1